From 59959e15946ed72741ae49f1901e87155f3efe77 Mon Sep 17 00:00:00 2001
From: Heinz Wrobel <Heinz.Wrobel@nxp.com>
Date: Mon, 20 May 2019 09:26:07 +0200
Subject: [PATCH] ls2084ardb_defconfig: Config was broken and U-Boot didn't
 compile

In prior updates, the configuration for the ls2084ardb (socketed RDB
with current silicon) wasn't pulled up along the BB Mini config.
This lead to the compilation of U-boot failing
This patch merges the necessary config bits back into the config
for 2017.12 to properly compile.

Signed-off-by: Heinz Wrobel <Heinz.Wrobel@nxp.com>
---
 configs/ls2084ardb_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/ls2084ardb_defconfig b/configs/ls2084ardb_defconfig
index a42c9ba..4feb286 100644
--- a/configs/ls2084ardb_defconfig
+++ b/configs/ls2084ardb_defconfig
@@ -3,6 +3,7 @@ CONFIG_TARGET_LS2080ARDB=y
 CONFIG_FSL_LS_PPA=y
 CONFIG_DEFAULT_DEVICE_TREE="fsl-ls2084a-rdb"
 # CONFIG_SYS_MALLOC_F is not set
+CONFIG_ENV_IS_IN_FLASH=y
 CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
 CONFIG_OF_BOARD_SETUP=y
@@ -30,7 +31,9 @@ CONFIG_DM=y
 CONFIG_FSL_CAAM=y
 CONFIG_MTD_NOR_FLASH=y
 CONFIG_DM_SPI_FLASH=y
+CONFIG_PHYLIB=y
 CONFIG_NETDEVICES=y
+CONFIG_PHY_GIGE=y
 CONFIG_E1000=y
 CONFIG_PCI=y
 CONFIG_DM_PCI=y
-- 
1.8.3.1

